### DISCUSSION TITLE
magit bookmarks!

### DISCUSSION THREAD

**Discussion by jdtsmith** [alphapapa/activities.el#111] [General]
Recently I learned that magit actually has had comprehensive bookmark support for a long time, but [lacked the auto-loads](https://github.com/magit/magit/discussions/5168) to have it work out of the box with activities.  An update is coming, and in the meantime you can:

```elisp
(use-package magit-section
  :autoload magit--handle-bookmark
  :config (require 'magit-bookmark))
```

**Comment by alphapapa** [comment]
For whatever reason, in my configuration, Magit gets loaded and the bookmarks have always worked for me.  But I'm glad you found and are fixing this.  Thanks.

**Comment by jdtsmith** [comment]
Perhaps you do not defer loading magit?

**Comment by jdtsmith** [comment]
[Merged](https://github.com/magit/magit/pull/5169#issuecomment-2221524540)
